docs/install/virtual_machine.md

```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
 
-
-
-
-
-
-
+With zeppelin running, an R Tutorial notebook will be available.  The R packages required to run the examples and graphs in this tutorial notebook were installed by this virtual machine.
```

Comment:
yah, we could do that for sure.
The package names for Python are listed on line 96 (earlier), not toward the end where you commented.
We can include in the same location if that makes sense.